src/Promact.TestCaseManagement.DomainModel/Models/Project.cs

```diff
@@ -20,8 +16,10 @@ public class Project : TestCaseManagementBase
 
         public bool IsActive { get; set; }
```

Comment:
There are 3 states of the Nullable Boolean

1. Null - Which means false but does not occupy any storage space
2. true - which means condition is true
3. false - this value might never be entered or will be entered if deleted project is reactivated later.

So, as you can see Nullable makes sense in practical programming.